src/components/AppBar/CustomAppBar.js

```diff
@@ -44,7 +52,9 @@ const CustomAppBar = (props) => {
     return (
         <AppBar position="static" className={classes.appBar}>
             <Toolbar variant="dense">
-                {isMobileScreen ? <MobileLogo height={32} /> : <Logo height={32} />}
+                <Suspense fallback={<div className={classes.fallback}>Loading...</div>}>
+                    {isMobileScreen ? <MobileLogo height={32} /> : <Logo height={32} />}
+                </Suspense>
```

Comment:
The jump here is a bit jarring... 
The width of "Loading..." causes the buttons to compress and just looks a little strange in the bar.

![appbar_load](https://user-images.githubusercontent.com/29494270/178864702-9da26929-7620-4db9-b3d7-b7fe090bd00e.gif)

The logo svg is pretty small, do we even need to lazy load it?

Reply:
Ooh yeah that's something that needs fixing for sure, I didn't even notice it since my internet is so fast :sweat_smile: Something about living in darkness by going faster than light, et cetera.

As for lazy loading logos: that was one of the goals Eric mentioned in #403, so I'll defer to him on that.

Reply:
> I didn't even notice it since my internet is so fast 😅  

In your chrome dev tools, you can go to the network tab and throttle your internet speed to slow things down
![throttle](https://user-images.githubusercontent.com/29494270/179035840-628e3f1e-f9c7-462a-abca-fd6bf139d929.gif)

Reply:
> As for lazy loading logos: that was one of the goals Eric mentioned in #403, so I'll defer to him on that.
I put it on the issue because it looked like a good place to start. If it's going to be difficult to fix that rendering problem then we should remove the lazy loading, because it's such a small asset it only contributes a few kilobytes to the bundle size.

Reply:
After thinking about it more, we should definitely not lazy load these images. Doing so adds extra complexity for almost no benefit.